src/write_http.c

```diff
@@ -80,6 +84,7 @@ struct wh_callback_s {
   pthread_mutex_t send_lock;
 
   int data_ttl;
+  char *metrics_prefix;
```

Comment:
Don't worry about this one, it can stay a `char *metric_prefix`.

Reply:
Great, I'll just modify the format files and leave this one untouched then.